src/chi/javascript/components/drawer.js

```diff
@@ -50,12 +51,32 @@ class Drawer extends Component {
       self.toggle();
     };
 
-    this._closeClickEventListener = function() {
+    this._closeClickEventListener = function(e) {
+      e.preventDefault();
```

Comment:
@davidojedacl There should be no need to prevent default. Please remove and check it still works.

Reply:
done